sdk/python/kfp/_client.py

```diff
@@ -136,6 +140,14 @@ def _load_config(self, host, client_id, namespace):
       config.host = os.path.join(config.host, Client.KUBE_PROXY_PATH.format(namespace))
     return config
 
+  def _is_iap_host(self, host, client_id):
+    if host and client_id:
+      return re.match(r'\S+.endpoints.\S+.cloud.goog', host)
+    return False
+
+  def _is_inverse_proxy_host(self, host):
+    return re.match(r'\S+dot-datalab-vm\S+.googleusercontent.com', host)
```

Comment:
Is it always `datalab-vm`?

Reply:
yes. and if we change later we can match against an array.